sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Release History
+
+## 12.5.0-preview.2 (2020-06)
+- Added support for service version 2019-12-12.
+- Added support for Blob Tags.
+- Added support for Blob Version.
+- Added support for Blob Container Soft Delete.
+- Added support for Object Replication Service.
```

Comment:
Should we include that for now it's just for OR Get Properties and Download Responses? (aka we have not added in the list blobs OR metadata yet)

Reply:
I think it might be ok, maybe we can mention it later when we add List Blobs ORS metadata support?  Or I could re-word this line to be more specific.

Reply:
Yeah I think we can leave it for now.